geometry_msgs/Vector3 rpos
float64[] drone_state
geometry_msgs/Vector3 rheading
float32[4] time_encoding